### PR TITLE
fix: Gives the wallet table some breathing room.

### PR DESCRIPTION
**Pull request by benkolera**
Was forcing a scrollbar, and shouldn't be up against the side of the page anyway.

### CODE REVIEW

File: frontend/src/Frontend/UI/Wallet.hs [hunk 1]
```diff
@@ -127,16 +127,16 @@ uiKeyItems model = do
     keyMap' = model ^. wallet_accounts
     keyMap = Map.fromAscList . IntMap.toAscList <$> keyMap'
     tableAttrs =
-      "style" =: "table-layout: fixed; width: 100%"
+      "style" =: "table-layout: fixed; width: 98%"
```
Comment (alexfmpe) [inline frontend/src/Frontend/UI/Wallet.hs:130]: How about leaving it at 100% and use a fixed margin?

File: backend/sass/common.blocks/_wallet.scss [hunk 2]
```diff
@@ -19,12 +19,11 @@
...
 .wallet__table-heading {
   @extend .table__heading;
-  padding: $medium-padding;
-  padding-bottom: $bigger-padding;
+  padding: 15px 30px 30px 15px;
 }
 
 .wallet__table-cell {
-  padding: $medium-padding;
+  padding: 15px 30px 15px 15px;
```
Comment (alexfmpe) [inline backend/sass/common.blocks/_wallet.scss]: Right padding is greater than left padding?

File: frontend/src/Frontend/UI/Wallet.hs (hunk 1 above)
Comment (benkolera) [reply frontend/src/Frontend/UI/Wallet.hs:130]: Something about it being at 100% was causing a scrollbar. And it's too late to figure out exactly why. It's more of a workaround to get rid of the scrollbar than just a margin.

File: backend/sass/common.blocks/_wallet.scss (hunk 2 above)
Comment (benkolera) [reply backend/sass/common.blocks/_wallet.scss]: Yeah. The rejected ticket was explicitly mentioning the fact that there wasn't enough padding between the keys. This way being lazy to get that effect, but you are right that it is confusing and I've done it a better way in 83dce35

**Pull request merged**